modules/node-tuning-hosted-cluster.adoc

```diff
@@ -0,0 +1,134 @@
++
+[source,yaml]
+----
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: tuned-1
+      namespace: clusters
+    data:
+      tuned: |
+        apiVersion: tuned.openshift.io/v1
+        kind: Tuned
+        metadata:
+          name: tuned-1
+          namespace: openshift-cluster-node-tuning-operator
+        spec:
+          profile:
+          - data: |
+              [main]
+              summary=Custom OpenShift profile
+              include=openshift-node
+              [sysctl]
+              vm.dirty_ratio="55"
+            name: tuned-1-profile
+          recommend:
+          - priority: 20
+            profile: tuned-1-profile
+----
++
+[NOTE]
+====
+If you do not add any labels to an entry in the `spec.recommend` section of the tuned spec, node-pool-based matching is assumed, so the highest priority profile in the `spec.recommend` section is applied to nodes in the pool. Although you can achieve more fine-grained node-label-based matching by setting a label value in the tuned `.spec.recommend.match` section, node labels will not persist during an upgrade unless you set the node pool `.spec.management.upgradeType` value to `InPlace`.
+====
+
+. Create the config map in the management cluster:
++
+[source, terminal]
+----   
+$ oc --kubeconfig="$MGMT_KUBECONFIG" create -f tuned-1.yaml
+----
+
+. Reference the config map in the node pool's `spec.tunedConfig` field, either by editing a node pool or creating one. In this example, assume that you have only one node pool, `nodepool-1`, which contains 2 nodes.
++
+[source,yaml]
+----
+    apiVersion: hypershift.openshift.io/v1alpha1
+    kind: NodePool
+    metadata:
+      ...
+      name: nodepool-1
+      namespace: clusters
+    ...
+    spec:
+      ...
+      tunedConfig:
+      - name: tuned-1
+    status:
+    ...
+----
++
+[NOTE]
+====
+You can reference the same config map in multiple node pools. In hosted control planes, the Node Tuning Operator appends a hash of the node pool name and namespace to the name of the tuned custom resources (tuneds) to distinguish them. Outside of this case, do not create multiple tuned profiles of the same name in different tuneds for the same hosted cluster.
+====
+
+.Verification
+
+Now that you have created the config map that contains a tuned manifest and referenced it in a node pool, the Node Tuning Operator syncs the tuned objects into the hosted cluster. You can verify which tuneds are defined and which profiles are set for each node.
+
+. List the tuned objects in the hosted cluster:
++
+[source,terminal]
+----
+$ oc --kubeconfig="$HC_KUBECONFIG" get Tuneds -n openshift-cluster-node-tuning-operator
+----
++
+.Example output
+[source,terminal]
+----
+    NAME       AGE
```

Comment:
One FINAL comment  - lines 97-100 are still indented. Everything else looks great!!

Reply:
Good catch! I'll fix that now.